crates/harp/src/modules/format.R

```diff
@@ -9,7 +9,7 @@
 # hard to recover from failed assumptions, including by unclassing and
 # reformatting with the default method.
 harp_format <- function(x, ...) {
-    if (is.object(x)) {
+    if (is.object(x) && !inherits(x, "formula")) {
         format_oo(x, ...)
```

Comment:
Since a formula _is_ an object, I think I'd prefer teaching `format_oo()` how to handle formulas

i.e. at the top of `format_oo()` maybe we can just add this special handling of

```r
if (inherits(x, "formula")) {
  # Need special handling for this scalar type
  return(base::format(x, ...))
}
```